websites/T/Threads/metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "241929388403195914",
+		"name": "Vasilis"
+	},
+	"service": "Threads",
+	"description": {
+		"en": "Say more with Threads — Instagram’s text-based conversation app, where communities come together to discuss everything from the topics you care about today to what’ll be trending tomorrow.",
+		"el_GR": "Πείτε περισσότερα με το Threads — την εφαρμογή συνομιλίας με κείμενο του Instagram, όπου οι κοινότητες συγκεντρώνονται για να συζητήσουν τα πάντα, από τα θέματα που σας ενδιαφέρουν σήμερα μέχρι αυτά που θα είναι μοντέρνα αύριο."
+	},
+	"url": "threads.net",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/qR37ayt.png",
```

Comment:
Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).